setup/web.sh

```diff
@@ -122,6 +122,16 @@ cat conf/mozilla-autoconfig.xml \
 	 > /var/lib/mailinabox/mozilla-autoconfig.xml
 chmod a+r /var/lib/mailinabox/mozilla-autoconfig.xml
 
+# Create a generic mta-sts.txt file which is exposed via the
+# nginx configuration at /.well-known/mta-sts.txt
+# more documentation is available on: 
+# https://www.digitalocean.com/community/tutorials/how-to-configure-mta-sts-and-tls-reporting-for-your-domain-using-apache-on-ubuntu-18-04
+
+cat conf/mta-sts.txt \
+        | sed "s/PRIMARY_HOSTNAME/$PRIMARY_HOSTNAME/" \
+         > /var/lib/mailinabox/mta-sts.txt
```

Comment:
UUOC: A bash heredoc would be great here (and would eliminate the need for conf/mta-sts.txt  Checkout the previous PR for mta-sts: https://github.com/mail-in-a-box/mailinabox/pull/1556/files

Reply:
Ik dit this approach by purpose, because the other elements in setup/web/sh have the same format/approach.

The reason, I made this decision, was to keep the style the same everywhere.

By the way, thanks for the other PR for MTA-STS. I thought, I was the first one.